news/2021-12-02-skinning-contest-tides-of-winter-accouncement.md

```diff
@@ -0,0 +1,89 @@
+---
+layout: post
+title: "Skinning Contest: Tides of Winter"
+date: 2021-12-02 12:00:00 +0000
```

Comment:
collides with https://github.com/ppy/osu-wiki/pull/6467

Reply:
@Ephemeralis

Reply:
fwiw, as long as it doesn't collide with FA newsposts, it should be fine.

Reply:
exhibit A:
![image](https://user-images.githubusercontent.com/36758269/144095070-84b4c965-6687-454c-9417-577d0c14153f.png)

exhibit B:
![image](https://user-images.githubusercontent.com/36758269/144095185-dabf2725-8482-436b-a17e-9b6c472ab543.png)

maybe we're fine, idk

Reply:
it maybe is fine, but its still bad to bump someone from the topspot of the homepage when there are other open spots. The newspost goes too quickly thanks to FA still not being split off (https://github.com/ppy/osu-web/issues/7685), best to prevent collides wherever possible. Tho that being said, pushing it by a day would mean closing it on the 31st, which does suck, so guess people would only get 27days instead of 28 since we would still need to close on the 30th

Reply:
I'd say those are either an exception or an oversight, but let's wait for eph to chime in and give us a clear answer.

Reply:
clipping on non FA stuff is fine, though ideally avoided

we can probably redate this to roughly 12 hours from now, don't believe anything else is fighting for the slot